# Clustering DBSCAN
clustering_eps: 0.5
clustering_min_samples: 10

# Detect apriltag
tag_family: "tag36h11"

# Detect pointcloud
# pcd segment plane
segment_dist_thres: 0.02
segment_ransac_n: 3
segment_num_iterations: 10000

# Ransac 1:
ransac1_min_samples: 2
ransac1_res_thres: 0.02
ransac1_max_trials: 5000

# Ransac 2:
ransac2_min_samples: 2
ransac2_res_thres: 0.02
ransac2_max_trials: 5000

# Geometric calibration
reproj_error: 8
intensity_thres: 200
distance_from_prev: 100
horizontal_dimension: 3840
vertical_dimension: 2160
grid_horizontal_division: 5
grid_vertical_division: 5